bin/config.ts

```diff
@@ -52,6 +52,16 @@ export function getConfig(): SystemConfig {
           provider: "bedrock",
           name: "amazon.titan-embed-text-v1",
           dimensions: 1536,
+        },
+        {
```

Comment:
please include [titan](https://github.com/aws-samples/aws-genai-llm-chatbot/pull/359/files#diff-45deac45daa48ef2d176c1d42cb9629f2607b020c8ea4bd8ab6763ac6033ac5fR37) from https://github.com/aws-samples/aws-genai-llm-chatbot/pull/359 which i closed in favor of this PR

Reply:
Added

Reply:
Actually one thought I had on this @bigadsoleiman is we are using Unstructured to to handle ingestion and partition all content, or at least I think it is. Would this conflict with Unstructured partition_image with ocr? I'll look into it and see if there is overlap on functions.